etc/flux.service.in

```diff
@@ -3,11 +3,23 @@ Description=Flux message broker
 
 [Service]
 Environment=FLUX_USERDB_OPTIONS=--default-rolemask=user
-ExecStart=@X_BINDIR@/flux broker -Srundir=@X_RUNSTATEDIR@/flux -Slocal-uri=local://@X_RUNSTATEDIR@/flux/local -Sboot.method=config sleep inf
+TimeoutStopSec=90
+KillMode=mixed
+ExecStart=@X_BINDIR@/flux broker \
+  -Srundir=@X_RUNSTATEDIR@/flux \
+  -Slocal-uri=local://@X_RUNSTATEDIR@/flux/local \
+  -Sboot.method=config \
+  -Slog-stderr-level=6 \
+  -Scontent.backing-path=@X_LOCALSTATEDIR@/lib/flux/content.sqlite \
+  -Sbroker.rc2_none
 User=flux
 Group=flux
 RuntimeDirectory=flux
 RuntimeDirectoryMode=0755
+PermissionsStartOnly=true
```

Comment:
In looking up what this does, I noticed that `PermissionsStartOnly` is [deprecated](https://github.com/systemd/systemd/pull/10802) in systemd.

Apparently, they felt that adding super [special executable prefixes](https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStart=) was a better way to go.

However, `PermissionsStartOnly` [hasn't been removed yet](https://github.com/systemd/systemd/pull/10802#issuecomment-439446299), so I don't think we should change anything here unless you feel like it.

Reply:
Let's catch that another time.  Thanks!